static/src/js/app/index.js

```diff
@@ -1,6 +1,8 @@
 import Vue from 'vue';
 import VueKeybindings from 'vue-keybindings';
 import VueFilterDateFormat from '@vuejs-community/vue-filter-date-format';
+import VueGoodTablePlugin from 'vue-good-table';
+import 'vue-good-table/dist/vue-good-table.css';
```

Comment:
should this be imported in the css section of `PersonalVocab.vue`? Not sure what is most idiomatic.

Reply:
I'm not entirely sure. Since the `vue-good-table` component can be used anywhere, it seemed like it should be a global style instead of scoped to `PersonalVocab.vue`. But if it's only going to be used in that one component, then we should probably move it there. For now, I'm leaning toward keeping the two imports together in the `index.js`.

Reply:
👍